demo-apps/cf-extplugtest-server/src/main/java/org/eclipse/californium/extplugtests/ExtendedTestServer.java

```diff
@@ -60,7 +61,8 @@ public void applyDefaults(NetworkConfig config) {
 			config.setInt(Keys.MAX_RESOURCE_BODY_SIZE, DEFAULT_MAX_RESOURCE_SIZE);
 			config.setInt(Keys.MAX_MESSAGE_SIZE, DEFAULT_BLOCK_SIZE);
 			config.setInt(Keys.PREFERRED_BLOCK_SIZE, DEFAULT_BLOCK_SIZE);
-			config.setInt(Keys.EXCHANGE_LIFETIME, 24700); // 24.7s instead of 247s
+			config.setInt(Keys.EXCHANGE_LIFETIME, 24700); // 24.7s instead of
+															// 247s
```

Comment:
(a bit out of topic :/)
Do we have an issue with our formatter ? 
(It seems we have a 120 width line limit for code and 80 width line for comments and so this make some strange behavior like this for inline comments)